File: multiplication/src/main/java/learn/mssb/mssbmultiplication/challenge/ChallengeService.java
package learn.mssb.mssbmultiplication.challenge;

import java.util.List;

public interface ChallengeService {

    ChallengeAttempt verifyAttempt(ChallengeAttemptDto attempt);

    List<ChallengeAttempt> getStatsForUser(String userAlias);
}

File: multiplicationkt/src/main/kotlin/challenge/ChallengeService.kt
package learn.mssbkt.multiplicationkt.challenge

interface ChallengeService {
    /**
     * Проверяет, является ли правильной попытка пользователя [attemptDTO]
     */
    fun verifyAttempt(attemptDTO: ChallengeAttemptDTO): ChallengeAttempt

    fun getStatsForUser(alias: String): List<ChallengeAttempt>
}
